paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -638,6 +638,7 @@ internal class PlaygroundTestDriver(
 
         pressBuy()
 
+        button.waitForEnabled()
```

Comment:
EPMs were flaky after this change. I think it's because moving intent result observation outside of the confirmation start called freed up the Main dispatcher earlier, causing the EPM button click (for Fawry success, fail, and cancel) to be executed earlier than expected. I added a `wait` call here to wait until they become available.